### DISCUSSION TITLE
RPi 3B+ - Python error: can't initialize time

### DISCUSSION THREAD

**Discussion by ahjcarroll** [mealie-recipes/mealie#301] [Show and tell]
Firstly can I say, Hayden, Mealie is a great application and I'm really impressed at the speed of it's development.  And I'm very glad you decided to open it up to allow us all to benefit from your hard work. 

I installed it on a Raspberry 3B+, but it wouldn't run, I kept getting this error: 

                              rpi docker run error fatal python error: init_interp_main: can't initialize time.  

I then tried it on my Windows 10 desktop running Docker, and Mealie worked as expected, I also tried it on a Raspberry 3B running Ubuntu 20.04 and it worked there also.  So after some digging around, I found the solution here: https://docs.linuxserver.io/faq.  It appears to a bug in the libseccomp2 library for Raspbian (Debian).  So for anyone else out there trying to run Mealie on a Raspian and receiving this error, try:

$ wget http://ftp.us.debian.org/debian/pool/main/libs/libseccomp/libseccomp2_2.4.4-1~bpo10+1_armhf.deb
$ sudo dpkg -i libseccomp2_2.4.4-1~bpo10+1_armhf.deb

Now Mealie works perfectly on my rpi 3B+.

Thanks again, enjoy a coffee (or a beer) or 2 on me.

Andrew

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened; if you're still having deployment trouble, please feel free to re-open.

I'm currently going through some old discussions, so apologies for the necro.
